cd_scripts/e2e_starter_script.sh

```diff
@@ -0,0 +1,88 @@
+#! /bin/bash
```

Comment:
nit: can we remove "starter_script" from the script name ? and just name it like e2e_tests and install_tests ?